packages/wallet/src/W3mFrameProvider.ts

```diff
@@ -735,8 +800,11 @@ export class W3mFrameProvider {
     W3mFrameStorage.set(W3mFrameConstants.SOCIAL_USERNAME, username)
   }
 
-  private setEmailLoginSuccess(email: string) {
-    W3mFrameStorage.set(W3mFrameConstants.EMAIL, email)
+  private setEmailLoginSuccess(email?: string | null) {
+    if (email) {
+      W3mFrameStorage.set(W3mFrameConstants.EMAIL, email)
+    }
+
```

Comment:
Do we want to keep this or add a new one?
I mean you're technically not logging in with email 🤔

Maybe it's a matter of renaming?